src/createElement.js

```diff
@@ -0,0 +1,12 @@
+import { sanitizeAttr, sanitizeAttrVal, sanitizeElementName } from './sanitize';
+
+export default function createElement(element, attributes, children = '') {
+  const sanitizedElement = sanitizeElementName(element);
+  const attributesString = attributes ? ` ${Object.entries(attributes).map(([attr, val]) => {
+    if (val == null) return '';
+    if (val === true) return sanitizeAttr(attr);
+    if (val === false) { return ''; }
+    return `${sanitizeAttr(attr)}="${sanitizeAttrVal(val)}"`;
+  }).filter(Boolean).join(' ')}` : '';
+  return `<${sanitizedElement}${attributesString}>${children}</${sanitizedElement}>`;
```

Comment:
if the above change is made; this should probably use `sanitizeJSON` whenever `children` isn't a string?